pages/runtime/runtime.md

```diff
@@ -1,21 +1,83 @@
+
+Inside the container we provide a number of `RESIN_` namespaced environment variables. Below is a short description of some of these.
+
+|    Variable   	| Description 	|
+|:----------:	    |:-----------:	|
+| `RESIN_DEVICE_UUID` 	      |  The unique identification number for the device. This is used to identify it on resin.io	|
+| `RESIN_APP_ID` 	            |  ID number of the resin.io application the device is associated. 	|
+| `RESIN` 	                  |  The `RESIN=1` variable can be used by your software to detect that it is running on a resin.io device. 	|
+| `RESIN_SUPERVISOR_VERSION` 	|  The current version of the supervisor agent running on the device.	|
+| `RESIN_SUPERVISOR_API_KEY` 	|  Authentication key for the supervisor API. This makes sure requests to the supervisor are only coming from containers on the device. See the [Supervisor API reference][supervisor-api-link]	for detailed usage.|
+| `RESIN_SUPERVISOR_ADDRESS` 	|  The network address of the supervisor API. Default: `http://127.0.0.1:48484`	|
+| `RESIN_SUPERVISOR_HOST` 	  |  The IP address of the supervisor API.	Default: `127.0.0.1`|
+| `RESIN_SUPERVISOR_PORT` 	  |  The network port number for the supervisor API. Default: `48484`	|
+| `RESIN_API_KEY` 	          |  API key which can be used to authenticate requests to the resin.io backend. Can be used with resin SDK on the device. **WARNING** This API key gives the code full user permissions, so can be used to delete and update anything as you would on the Dashboard.  	|
+| `RESIN_DEVICE_RESTART` 	    |  This is a internal mechanism for restarting containers and can be ignored as its not very useful to application code.  Example: `1.13.0`	|
+
+Here's an example from a Raspberry Pi 3:
+```
+root@raspberrypi3-cb6f09d:/# printenv | grep RESIN
+RESIN_SUPERVISOR_API_KEY=1111deadbeef2222
+RESIN_APP_ID=116522
+RESIN=1
+RESIN_SUPERVISOR_ADDRESS=http://127.0.0.1:48484
+RESIN_DEVICE_RESTART=3505733431986444
+RESIN_SUPERVISOR_HOST=127.0.0.1
+RESIN_DEVICE_UUID=cb6f09d18ab4c08556f54a5bd7cfd353d4907c4a61998ba8a54cd9f2abc5ee
+RESIN_API_KEY=deadbeef12345
+RESIN_SUPERVISOR_VERSION=1.13.0
+RESIN_SUPERVISOR_PORT=48484
+```
+
+## Persistent Storage		
+
+{{> "general/persistent-storage"}}
 
-# Runtime Environment
 
 ## Exposed Ports
 
 Resin.io devices expose all ports by default, meaning you can run applications
 which listen on any port without issue. There is no need to have the docker `EXPOSE` command in your `Dockerfile`.
 
-## Device URLS
+## Public Device URLS
 
 Resin.io currently exposes port 80 for web forwarding. To enable web forwarding on a specific device, navigate to the device's **actions** tab on the resin.io dashboard and select the `Enable a public url for this device` checkbox. For more information about device URLS you can head over to the [Device Management Page](/management/devices#enable-public-device-url)
```

Comment:
Actually port 8080 is exposed as well, but only on http connection.

* port 80 exposed at: `https://<UUID>.resindevice.io` and `http://<UUID>.resindevice.io`
* port 8080 exposed at: `http://UUID.resindevice.io:8080`

Thus can have two distinct applications listening. The UI does not give any indication to this, though, just mentioned in chat.